### PR TITLE
Translation: JavaScript Fundamentals > Operators

### PR DESCRIPTION
**Pull request by Zim123**
https://github.com/javascript-tutorial/uk.javascript.info/issues/16

### CODE REVIEW

File: 1-js/02-first-steps/07-operators/2-assignment-result/task.md [hunk 1]
```diff
@@ -2,9 +2,9 @@ importance: 3
 
 ---
 
-# Assignment result
+# Рузльтат присвоювання
```
Comment (tarasyyyk) [inline 1-js/02-first-steps/07-operators/2-assignment-result/task.md]: Результат

File: 1-js/02-first-steps/07-operators/article.md [hunk 2]
```diff
@@ -1,170 +1,170 @@
...
+# Оператори
 
-We know many operators from school. They are things like addition `+`, multiplication `*`, subtraction `-`, and so on.
+Ми знаємо багато операторів зі школи. Це такі речі, як додавання `+`, множення `*`, віднімання `-`, тощо.
 
-In this chapter, we'll concentrate on aspects of operators that are not covered by school arithmetic.
+У цьому розділі ми зупинимося на аспектах операторів, які не охоплені шкільною арифметикою.
 
-## Terms: "unary", "binary", "operand"
+## Тарміни: "унарний", "бінарний", "операнд"
```
Comment (tarasyyyk) [inline 1-js/02-first-steps/07-operators/article.md]: Терміни

File: 1-js/02-first-steps/07-operators/article.md [hunk 3]
```diff
@@ -1,170 +1,170 @@
...
 
-Before we move on, let's grasp some common terminology.
+Перш ніж ми почнемо, давайте розберемо певну загальну термінологію.
 
-- *An operand* -- is what operators are applied to. For instance, in the multiplication of `5 * 2` there are two operands: the left operand is `5` and the right operand is `2`. Sometimes, people call these "arguments" instead of "operands".
-- An operator is *unary* if it has a single operand. For example, the unary negation `-` reverses the sign of a number:
+- *Операнд* -- це те, до чого затосовуються оператори. Наприклад, у множенні `5 * 2` є два операнди: лівий операнд `5` і правий операнд `2`. Іноді люди називають їх "аргументами", а не "операндами".
+- Опреатор є *унарним*, якщо він має один операнд. Наприклад, унарне заперечення `-` змінює знак числа:
```
Comment (tarasyyyk) [inline 1-js/02-first-steps/07-operators/article.md]: Оператор

File: 1-js/02-first-steps/07-operators/2-assignment-result/task.md (hunk 1 above)
Comment (tarasyyyk) [reply 1-js/02-first-steps/07-operators/2-assignment-result/task.md]: Also, we need to disqus word "Assignment". There are two possible variants: "присвоювання" and "присвоєння". I used the last one in my translations, but not sure which one is better.

Comment (Zim123) [reply 1-js/02-first-steps/07-operators/2-assignment-result/task.md]: Good question. I also had a doubt about this, and somewhy choose my options, but now I checked 
http://sum.in.ua/s/prysvojennja
https://uk.wikipedia.org/wiki/Присвоєння_(програмування)
So, your option makes more sense, I'll correct my article.

**Comment by Zim123** [comment]
Made all needed fixes.

**Comment by Zim123** [comment]
@tarasyyyk 
And I'm ready to take the next one :)

https://github.com/javascript-tutorial/uk.javascript.info/issues/20

**Pull request merged**